internal/service/httpserver.go

```diff
@@ -125,8 +133,64 @@ func (b *HttpServer) BootstrapHandler(
 			b.isRunning = false
 		}()
 
+		var ln net.Listener
+		listenMode := strings.ToLower(b.cfg.Service.SecurityOptions[bscfg.SecurityModeKey])
```

Comment:
all this functionality should also be moved to `go-mod-bootstrap` or some other common place. this code is largely simlar to code in [`app-functions-sdk-go/internal/webserver/server.go`](https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/internal/webserver/server.go#L133-L179)

Reply:
yes, we should do it when we have a chance